### PR TITLE
Add nbqa to pre-commit config

### PR DESCRIPTION
**Pull request by StanczakDominik**
- [ ] I have added a changelog entry for this pull request.

- [ ] If adding new functionality, I have added tests and
      docstrings.

- [ ] I have fixed any newly failing tests.

### CODE REVIEW

**Comment by namurphy** [comment]
Interesting!  I haven't used [nbqa](https://nbqa.readthedocs.io/en/latest/index.html) before but it seems really helpful.

File: docs/notebooks/cold_plasma_tensor_elements.ipynb [hunk 1]
```diff
@@ -287,7 +304,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.2"
+   "version": "3.9.1"
```
Comment (namurphy) [inline docs/notebooks/cold_plasma_tensor_elements.ipynb:307]: This line makes me happy.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -22,3 +22,12 @@ repos:
...
     -   id: black
+- repo: https://github.com/nbQA-dev/nbQA
+  rev: 0.5.7
+  hooks:
+    - id: nbqa-black
+      additional_dependencies: [black==19.10b0]
+      args: [--nbqa-mutate]
+    - id: nbqa-isort
+      additional_dependencies: [isort==5.7.0]
+      args: [--nbqa-mutate]
```
Comment (namurphy) [inline .pre-commit-config.yaml:33]: Again, this is nice readable code.

**Review by namurphy — APPROVED** [review]
Looks like nbqa had lots of tiny fixes but this looks like it's going to be helpful in making our notebooks nice and consistent.  Thank you again for this and #996!

**Comment by StanczakDominik** [comment]
https://github.com/PlasmaPy/PlasmaPy/runs/1792630778 shows just about how it'd show up before committing if actually using pre-commit (as I keep telling people to... :sweat_smile: ), or if a notebook has errors. It's probably easiest to apply those fixes by installing pre-commit and running it :D

**Comment by StanczakDominik** [comment]
Anyway, my point is: if you're fine with how this outputs, feel free to merge :)

**Pull request merged**